lib/sailor.js

```diff
@@ -337,6 +338,7 @@ class Sailor {
                 }, 'processMessage emit error');
                 headers.end = new Date().getTime();
                 const props = createAmqpProperties(headers);
+                props[Symbol.for('objectId')] = messageObjectId;
                 return self.amqpConnection.sendError(err, props, message.content, self.throttles.error);
             }
 
```

Comment:
I suggest to just pass message to sendError everywhere. This won't pollute properties with "hidden" unrelated symbols and less code will be written.